gensim/models/word2vec.py

```diff
@@ -941,20 +950,26 @@ def _do_train_job(self, sentences, alpha, inits):
         work, neu1 = inits
         tally = 0
         if self.sg:
-            tally += train_batch_sg(self, sentences, alpha, work, self.compute_loss)
+            tally += train_batch_sg(
```

Comment:
This previously fit on one line. What is the benefit of this change?

Same thing for the below.

Reply:
Sorry, this comes from an intermediate version when a parameter had been added, which has then been deleted again. I will restore the previous version (same below).